index.template.html

```diff
@@ -3916,6 +3925,115 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
 
   </section>
 
+  <section id="thing-templates" class="normative">
+  <h1>Thing Model</h1>
+      
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
+    driven in <a href="#sec-vocabulary-definition"></a> and <a href="#sec-td-serialization"></a>. Thing Model samples can 
+    also be seen as a subset of the <a>Thing Descriptions</a> information model, which may not contain, 
+    for example, concrete protocol and security information.
+    </p>
+    <p>
+    The Thing Model enables:
+    <ul>
+    <li> onboard and manage of multiple <a>Thing</a> models, e.g., by a cloud service. </li>
+    <li> simulate of devices/<a>Things</a> that have not yet been developed.</li>
+    <li> develop common applications across devices from different manufacturers that share a common <a>Thing</a> model.</li>
+    <li> combining multiple models into a <a>Thing</a>.
+    </ul>
+    </p>
+    <p>
+    The Thing Model is a logical description of the interface and possible interaction with <a>Thing's</a>
+    <a>Properties</a>, <a>Actions</a>, and <a>Events</a>, however it does not contain <a>Thing</a> instance-specific
+    information, such as security information and concrete protocol usage, or even a serial number and GPS location, .
+    </p><p>
+    Since a Thing Model MAY not contain complete <a>Protocol Binding</a> information such as endpoint address and/or
```

Comment:
Please check RFC 2119 for the usage of "MAY". Perhaps "MUST" would be appropriate here?

Reply:
from RFC 2119: MAY   This word, or the adjective "OPTIONAL", mean that an item is
   truly optional. 

I think this is the right way, because the implementer should be aware that a thing model gets along without or partly without this vocabulary. 

Maybe what would be also possible "SHOULD NOT". What do you think?

Reply:
In my new commit I used the term "SHOULD NOT".